### PR TITLE
README.md file updated  

### PR DESCRIPTION
**Pull request by anishsarangi**
README.md file is updated with the some more commands which will help the other developers to run this code in their machine locally without any difficulties.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,3 @@ functions/node_modules
 
 functions/firebase-debug.log
 npm-debug.log
```
Comment (hyzhak) [inline .gitignore]: please don't remove last empty new line. It is recommended by standards

File: firebase-debug.log [hunk 2]
```diff
@@ -0,0 +1,48 @@
+[debug] [2018-03-14T12:31:09.519Z] ----------------------------------------------------------------------
```
Comment (hyzhak) [inline firebase-debug.log]: we don't need this file in a repository. You can add it to `.gitignore`.

File: README.md [hunk 3]
```diff
@@ -5,7 +5,15 @@
 Run server local with colorful logs
 
 ```
+
+Install firebase globally
```
Comment (hyzhak) [inline README.md]: because it is the comment you should comment symbol before the comment

File: functions/package-lock.json [hunk 4]
```diff
@@ -7,7 +7,12 @@
     "@firebase/app": {
       "version": "0.1.10",
       "resolved": "https://registry.npmjs.org/@firebase/app/-/app-0.1.10.tgz",
+
```
Comment (hyzhak) [inline functions/package-lock.json]: I think you should revert all changes in this file

**Comment by anishsarangi** [comment]
Closing this PR .Soon requesting a new PR making the changes

**Pull request closed without merging**